docs/stores/vuex/vuex.md

```diff
@@ -428,3 +428,17 @@ Here's the complete example:
 - [Vuex](https://vuex.vuejs.org/) - Vuex official docs.
 
 - Additional examples - [Vuex](https://github.com/vuejs/vuex#examples)
+
+## Project Tasks
+
+You might have notice the emitter patter to be useful, but also, to be hard to maintain and implement as your application gets complex. Let's implement a  Vuex patter and see hot it comes into place:
+
+1. Add Vuex store as main state source.
+
+## Resolution
+
+Vuex is definitely a much more simpler approach when you need to communicate information between components, let's take a look a its implementation and benefits:
```

Comment:
missing in a "t" in:
 let's take a look **at** its implementation and benefits: